Docker/Centos8/Dockerfile

```diff
@@ -3,6 +3,11 @@ FROM centos:8
 LABEL project="centos8-gluster-build"
 LABEL maintainer "gluster-users@gluster.org"
 
+#Changes for pointing to Centos Stream repo
+RUN cd /etc/yum.repos.d/
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
```

Comment:
when do you use '|' and when '/' ?

Reply:
s is searching for the baseurl string pointing to mirror.centos.org and pointing to vault.centos.org.